consultation_analyser/consultations/jinja2/schema.html

```diff
@@ -0,0 +1,88 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">Consultation data schema v0.1</h1>
+
+  <p class="govuk-body">This page lists the data components required for a consultation to be suitable for processing by the analyser. </p>
+
+  <p class="govuk-body">Data should be provided in two JSON files: one for the consultation and one for its answers. Those two data files are described below.</p>
+
+  <h2 class="govuk-heading-l">Consultation JSON</h2>
+  <p class="govuk-body">The consultation file should describe the consultation conforming to this schema.</p>
+
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        JSON schema
+      </span>
+    </summary>
+    <div class="govuk-details__text">
+      <pre><code>{{ json_schemas['consultation'] }}</code></pre>
+    </div>
+  </details>
+
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        Example JSON
+      </span>
+    </summary>
+    <div class="govuk-details__text">
+      <pre><code>{{ json_examples['consultation'] }}</code></pre>
+    </div>
+  </details>
+
+  <h2 class="govuk-heading-l">ConsultationResponseList JSON</h2>
+  <p class="govuk-body">The consultation responses file should contain a list of consultation responses conforming to this schema.</p>
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        JSON schema
+      </span>
+    </summary>
+    <div class="govuk-details__text">
+      <pre><code>{{ json_schemas['consultation_response_list'] }}</code></pre>
+    </div>
+  </details>
+
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        Example JSON
+      </span>
+    </summary>
+    <div class="govuk-details__text">
+      <pre><code>{{ json_examples['consultation_response_list'] }}</code></pre>
+    </div>
+  </details>
+
+
+  <h2 class="govuk-heading-l">Entities</h2>
+  {% for schema in entity_schemas %}
+    <h2 class="govuk-heading-m">{{ schema.name() }}</h2>
```

Comment:
These sub-headings within "Entities" should be <h3>s

Reply:
Already fixed.